src/index.ts

```diff
@@ -11,18 +11,9 @@ import { UniqueMessageIdProvider } from './uniqueMessageIdProvider.js'
 import { ConnectOptions } from './interfaces/connectOptions.js'
 import { protocols } from './utils/constants.js'
 import { logger } from './utils/logger.js'
-import { handle } from './utils/error.js'
 
 import { URL } from 'url'
 
-process.on("unhandledRejection", (err) => {
-  throw err;
-})
-
-process.on("uncaughtException", (err) => {
-  handle(err);
-})
-

```

Comment:
We should not have this. If a user imports our package, then every single uncaught exception or unhandled rejection that happens on that Node process will be altered by us.